proto/iotics/api/meta.proto

```diff
@@ -101,13 +107,46 @@ message SparqlQueryResponse {
     // Content type of the result.
     SparqlResultType contentType = 5;
 
-    // Query result chunk, encoded according to actualType. The maximum size of each chunk is host-specific.
+    // Query result chunk, encoded according to actualType. 
+    // Note that:
+    // - The maximum size of each chunk is host-specific.
     bytes resultChunk = 6;
   }
 
   // Headers for the query result. clientRef within can be used to identify which query the result applies to.
   Headers headers = 1;
 
-  // SPARQL result chunk payload.
+  // SPARQL query result chunk payload.
+  Payload payload = 2;
+}
+
+// Performs a SPARQL update against custom metadata only.
+message SparqlUpdateRequest {
+
+  // SPARQL update request headers
+  Headers headers = 1;
+
+  // SPARQL update request payload.
+  message Payload {
+    // A UTF8-encoded SPARQL 1.1 update
+    // Note that:
+    // - The maximum size of each chunk is host-specific defaulting to 4MB (https://pkg.go.dev/google.golang.org/grpc#MaxRecvMsgSize);
+    // - The client should split large payloads into multiple updateChunks;
+    // - The query will only be regarded as complete once the send-direction of the stream has been closed. 
+    //   For example, in Golang - https://pkg.go.dev/google.golang.org/grpc#ClientStream.CloseSend
+    bytes updateChunk = 1;
+  }
+
+  // SPARQL update request payload.
   Payload payload = 2;
 }
+
+// Response of the SPARQL update request.
+message SparqlUpdateResponse {
+  // SPARQL update response headers
+  Headers headers = 1;
+
+  // Result error status. If set, this will indicate a problem with running the update (e.g. invalid syntax) 
+  // as opposed to a more general issue (in which case the standard gRPC error mechanism will be used and the stream terminated).
+  google.rpc.Status status = 2;
```

Comment:
No need for the status, it can be returned as a standard gRPC error. 
(we are using wrapped status error for the subscription services where we don't want to kill the subscription when an error occurs for a given request. Ex: search pub/sub and meta pub/sub -> for later)